app/backend/requirements.txt

```diff
@@ -5,3 +5,8 @@ openai[datalib]==0.27.8
 tiktoken==0.4.0
 azure-search-documents==11.4.0b6
 azure-storage-blob==12.14.1
+
+azure-monitor-opentelemetry --pre
```

Comment:
No version numbers?

Reply:
The version dependencies for Otel are tricky right now until there's a V1. It needs to be flexible. I could set an upper-limit incase there are more breaking changes

Reply:
Okay, thats fine, just add a comment about that. Upper limit is good if possible.